bash/execute_update_properties.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+python ../../python/update_package_properties.py --gh_token "${GITHUB_TOKEN}" --prj_name "${PRJ_NAME}" --prj_ver "${PRJ_VER}"  --tag_name "${TAG_NAME}" --fancy "${FANCY}" \
```

Comment:
Can we split lines like this into more lines where we have a single parameter on each line?

I agree with @JelteF, and this seems to be unused

Reply:
Removed the file